packages/commonjs/src/resolve-id.js

```diff
@@ -3,11 +3,12 @@ import { statSync } from 'fs';
 import { dirname, resolve, sep } from 'path';
 
 import {
+  DYNAMIC_JSON_PREFIX, DYNAMIC_PACKAGES_ID,
   getExternalProxyId,
   getIdFromProxyId,
   getProxyId,
   HELPERS_ID,
-  PROXY_SUFFIX
+  PROXY_SUFFIX,
```

Comment:
please remove trailing commas. not sure why linting didn't pick that up, but we don't have trailing commas enabled.